Dockerfile

```diff
@@ -8,70 +8,113 @@ ENV PASSWORD=password
 ENV LICENCE=notset
 ENV MODULE=ADS
 
+ARG DEBIAN_FRONTEND noninteractive
```

Comment:
This line is missing an `=` (`ARG DEBIAN_FRONTEND=noninteractive`) but I will fix that :)